apps/web/src/__tests__/Study.spec.tsx

```diff
@@ -174,6 +174,9 @@ describe('Study page', () => {
   jest.mocked(getServerSession).mockResolvedValue(userWithSponsorContactRole)
 
   test('Default layout', async () => {
+    const originalEnv = process.env.EDIT_STUDY_ALLOW_LIST
+    process.env.EDIT_STUDY_ALLOW_LIST = userWithSponsorContactRole.user?.email
```

Comment:
Modifying the process environment will be visible across multiple threads.
Will this cause intermittent failures when running tests in parallel?

Reply:
Nope this shouldn't affect any other tests.